doc/api/fs.md

```diff
@@ -4259,6 +4313,32 @@ through any other `fs` operation may lead to undefined behavior.
 
 See the POSIX close(2) documentation for more detail.
 
+### `fs.copySync(src, dest[, options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `src` {string|Buffer|URL} source path to copy.
+* `dest` {string|Buffer|URL} destination path to copy to.
+* `options` {Object|Function}
+  * `dereference` {boolean} dereference symlinks. **Default:** `false`.
+  * `errorOnExist` {boolean} when `force` is `false`, and the destination
+    exists, throw an error. **Default:** `false`.
+  * `filter` {Function} Function to filter copied files/directories. Return
```

Comment:
What happens here if the user passes an async function?

Reply:
A function would need to be synchronous for the synchronous version of filter to work, we could validate this in the sync version of the algorithm.

Reply:
Yeah, that needs to be called out in the docs and guarded against in the code.